src/server/services/wms/qgswmsrenderer.cpp

```diff
@@ -1089,6 +1089,13 @@ namespace QgsWms
     QgsMapSettings mapSettings;
     std::unique_ptr<QImage> outputImage( createImage( imageWidth, imageHeight ) );
 
+    // The CRS parameter is considered as mandatory in configureMapSettings
+    // but in the case of filter parameter, CRS parameter has not to be mandatory
```

Comment:
Can't we have a `BBOX` parameter at the same time as a `FILTER`?

Reply:
I think, but I don't know how QGIS Server used it.

I just want to avoid to add a CRS, when you just want to use a filter without bbox.

Reply:
> I just want to avoid to add a CRS, when you just want to use a filter without bbox.

In this case, I think you should check that the `BBOX` parameter is not defined too.

Reply:
Does the `filterGeom` need a BBOX and a CRS or only a CRS ?

Reply:
@pblottiere In the case of the use of FILTER parameter, the filterRect is set to an empty rectangle so spatial search is not done.